.github/workflows/tx-hammer-plugins.yml

```diff
@@ -0,0 +1,96 @@
+name: Translations
+
+on:
+  workflow_call:
+    inputs:
+      gem_name:
+        description: 'Name of the Gem'
+        required: true
+        type: 'string'
+    secrets:
+      tx_token:
+        description: 'API token for the Transifex.'
+        required: true
+      gh_token:
+        description: 'GitHub token with permissions to create a pull request'
+        required: true
+
+jobs:
+  tx-update:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.gem_name  }}
+
+    steps:
+      - name: Get hammer-cli
+        uses: actions/checkout@v4
+        with:
+          repository: theforeman/hammer-cli
+          ref: master
+          path: hammer-cli
+
+      - name: Get ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          path: ${{ inputs.gem_name  }}
+
+      - name: Install Transifex dependencies
+        run: |
+          sudo apt-get -y install gettext
+
+      - name: "Install Transifex"
+        run: |
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+          sudo mv tx /usr/local/bin/
+          sudo chown -R runner:runner /usr/local/bin
```

Comment:
I don't see a reason to chown it. Normally things in `/usr/local/bin` are `root:root` mode 0755 which works fine. In this particular case you already download it as `runner` and `mv` maintains that so if anything, I think this is redundant:
```suggestion
```